mistletoe/block_token.py

```diff
@@ -128,7 +128,12 @@ class Document(BlockToken):
     """
     Document token.
     This is a container block token. Its children are block tokens - container or leaf ones.
+
+    Attributes:
+        children (list): inner tokens.
```

Comment:
Sure, I can do that. This has gone back and forth a few times, as at first I thought I'd add _all_ the attributes, but then decided not to include the "unofficial" ones. So that sometimes left only the inherited attributes.